packages/mui-material/src/SpeedDial/SpeedDial.d.ts

```diff
@@ -83,15 +124,25 @@ export interface SpeedDialProps
    *   enter: theme.transitions.duration.enteringScreen,
    *   exit: theme.transitions.duration.leavingScreen,
    * }
+   * @deprecated
    */
   transitionDuration?: TransitionProps['timeout'];
   /**
    * Props applied to the transition element.
-   * By default, the element is based on this [`Transition`](https://reactcommunity.org/react-transition-group/transition/) component.
+   * By default, the element is based on this [`Transition`](http://reactcommunity.org/react-transition-group/transition/) component.
+   * @deprecated
    */
   TransitionProps?: TransitionProps;
+  /**
+   * The components used for each slot inside.
+   *
+   * @default {}
+   * @deprecated Use `slots.transition` instead. This prop will be removed in v7. [How to migrate](/material-ui/migration/migrating-from-deprecated-apis/)
+   */
```

Comment:
This should go above TransitionComponent

Reply:
yeah, i think i got confused an put the deprecated info under thinking that was related to the prop above, but its the other way around.